tests/load_tests/test_queue_dispatcher.py

```diff
@@ -0,0 +1,134 @@
+import multiprocessing
+import threading
+import time
+import uuid
+
+from sky import sky_logging
+from sky.server.requests import executor
+from sky.server.requests import payloads
+from sky.server.requests import process
+from sky.server.requests import requests as api_requests
+from sky.server.requests.queues import mp_queue
+
+logger = sky_logging.init_logger(__name__)
+
+
+def disable_logger() -> None:
+    server_logger = sky_logging.init_logger('sky.server')
+    server_logger.handlers.clear()
+    server_logger.propagate = False
+
+
+disable_logger()
+
+
+class MockExecutor(process.BurstableExecutor):
+
+    def submit_until_success(self, *args, **kwargs):
+        return None
+
+
+def dummy_func(*args, **kwargs):
+    pass
+
+
+def schedule():
+    worker = executor.RequestWorker(
+        schedule_type=api_requests.ScheduleType.LONG,
+        garanteed_parallelism=1,
+        burstable_parallelism=1)
+    mock_executor = MockExecutor(garanteed_workers=1, burst_workers=1)
+    queue = executor._get_queue(api_requests.ScheduleType.LONG)  # pylint: disable=protected-access
+    while not queue.queue.empty():
+        worker.process_request(mock_executor, queue)
+
+
+def benchmark_queue_dispatcher(queue_backend: str,
+                               worker_mode: str,
+                               count=1) -> float:
+    api_requests.reset_db_and_logs()
+    num_requests = 10000
+    executor.queue_backend = queue_backend
+
+    queue_server = None
+    if queue_backend == executor.QueueBackend.MULTIPROCESSING:
+        port = mp_queue.DEFAULT_QUEUE_MANAGER_PORT
+        queue_server = multiprocessing.Process(
+            target=mp_queue.start_queue_manager,
+            args=([api_requests.ScheduleType.LONG.value], port))
+        queue_server.start()
+        mp_queue.wait_for_queues_to_be_ready(
+            [api_requests.ScheduleType.LONG.value], queue_server, port=port)
+    elif queue_backend == executor.QueueBackend.LOCAL:
+        pass
+    else:
+        raise RuntimeError(f'Invalid queue backend: {queue_backend}')
+
+    def enqueue_requests():
+        for _ in range(num_requests):
+            body = payloads.StatusBody()
+            executor.schedule_request(
+                request_id=str(uuid.uuid4()),
+                request_name='status',
+                is_skypilot_system=True,
```

Comment:
should be `False` to mimic the user behavior?

Reply:
Good catch!